### PR TITLE
Leaderboard: final touches

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
## HTML/CSS requirements ♻️ 
 - Improve the look and feel of the application 👍🏽 
 - Keep the general layout of the wireframe 👍🏽 
- Use plain CSS or any CSS framework 👍🏽

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 > The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. 
 
+![screenshot](./app_screenshot.png)
 
 ## Built With
```
Comment (Whoistolu) [inline README.md:7]: - [ ] [OPTIONAL] In the getting started part, you actually stated how a user can get a copy of the project but you also need to state that the user needs to run `npm i` to get all the project dependencies and run `npm start` to run the project.

**Review by Whoistolu — APPROVED** [review]
### Hi @kinginthenorthcodez 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (kinginthenorthcodez) [reply README.md:7]: Thanks for making note of that, have added your suggestion. 👍🏽

**Comment by kinginthenorthcodez** [comment]
Merge #4 into development

**Pull request merged**